Dockerfile

```diff
@@ -0,0 +1,85 @@
+# Stage 1: Build Stage
+FROM rust:latest AS builder
+
+# Set the working directory inside the container
+WORKDIR /usr/src/app
+
+# Install necessary packages
+RUN apt-get update && apt-get install -y \
+    bash \
+    build-essential \
+    libssl-dev \
+    pkg-config \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install the necessary Rust target for ARM64 (Raspberry Pi 5)
+RUN rustup target add aarch64-unknown-linux-gnu
+
+# Copy the main workspace Cargo.toml and Cargo.lock to define workspace structure
+COPY Cargo.toml Cargo.lock ./
+
+# Copy each module's Cargo.toml to maintain the workspace structure
+COPY ./entity/Cargo.toml ./entity/Cargo.toml
+COPY ./entity_api/Cargo.toml ./entity_api/Cargo.toml
+COPY ./migration/Cargo.toml ./migration/Cargo.toml
+COPY ./service/Cargo.toml ./service/Cargo.toml
+COPY ./web/Cargo.toml ./web/Cargo.toml
+
+# Copy the complete source code into the container's working directory
+COPY . .
+
+# Set the target directory to ensure binaries are placed in a known location
+#ENV CARGO_TARGET_DIR=/usr/src/app/target/aarch64-unknown-linux-gnu/release
+
+# Build the project
+RUN cargo build --release
```

Comment:
```suggestion
RUN cargo build --release --workspace
```

@lmcdonough I figured out if you add —-workspace to the end you'll get the missing `migration` binary that line 55 is expecting and the Dockerfile succeeds in building then.